core/state/statedb.go

```diff
@@ -435,11 +435,12 @@ func (s *StateDB) SetNonce(addr common.Address, nonce uint64) {
 	}
 }
 
-func (s *StateDB) SetCode(addr common.Address, code []byte) {
+func (s *StateDB) SetCode(addr common.Address, code []byte) (prev []byte) {
```

Comment:
It seems like `prev` is not used

Reply:
for go-ethereum similarity https://github.com/ethereum/go-ethereum/blob/master/core/vm/interface.go#L46